tests/SprykerEcoTest/Zed/Unzer/Business/UnzerFacade/CreateUnzerCredentialsTest.php

```diff
@@ -0,0 +1,297 @@
+
+        // Act
+        $this->tester->getFacade()->createUnzerCredentials($unzerCredentialsTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCreateUnzerCredentialsThrowsNullValueExceptionWhileUnzerKeypairPrivateKeyIsMissing(): void
+    {
+        // Arrange
+        $this->tester->ensureUnzerCredentialsTableIsEmpty();
+        $unzerCredentialsTransfer = (new UnzerCredentialsBuilder([
+            UnzerCredentialsTransfer::ID_UNZER_CREDENTIALS => null
+        ]))
+            ->withUnzerKeypair([
+                UnzerKeypairTransfer::PRIVATE_KEY => null,
+            ])->build();
+
+        // Assert
+        $this->expectException(NullValueException::class);
+        $this->expectExceptionMessage('Property "privateKey" of transfer `Generated\Shared\Transfer\UnzerKeypairTransfer` is null.');
+
+        // Act
+        $this->tester->getFacade()->createUnzerCredentials($unzerCredentialsTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCreateUnzerCredentialsThrowsNullValueExceptionWhileTypeIsMissing(): void
+    {
+        // Arrange
+        $this->tester->ensureUnzerCredentialsTableIsEmpty();
+        $unzerCredentialsTransfer = (new UnzerCredentialsBuilder([
+            UnzerCredentialsTransfer::TYPE => null
+        ]))
+            ->withUnzerKeypair()
+            ->build();
+
+        // Assert
+        $this->expectException(NullValueException::class);
+        $this->expectExceptionMessage('Property "type" of transfer `Generated\Shared\Transfer\UnzerCredentialsTransfer` is null.');
+
+        // Act
+        $this->tester->getFacade()->createUnzerCredentials($unzerCredentialsTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCreateMarketplaceMainMerchantUnzerCredentialsThrowsNullValueExceptionWhileParentIdUnzerCredentialsIsMissing(): void
+    {
+        // Arrange
+        $this->tester->ensureUnzerCredentialsTableIsEmpty();
+        $marketplaceUnzerCredentials = $this->tester->haveMarketplaceUnzerCredentials();
+        $unzerCredentialsTransfer = (new UnzerCredentialsBuilder([
+            UnzerCredentialsTransfer::TYPE => UnzerSharedConstants::UNZER_CONFIG_TYPE_MARKETPLACE_MAIN_MERCHANT,
+            UnzerCredentialsTransfer::PARENT_ID_UNZER_CREDENTIALS => null,
+            UnzerCredentialsTransfer::STORE_RELATION => [
+                StoreRelationTransfer::STORES => [$marketplaceUnzerCredentials->getStoreRelation()->getStores()->offsetGet(0)],
+                StoreRelationTransfer::ID_STORES => [$marketplaceUnzerCredentials->getStoreRelation()->getStores()->offsetGet(0)->getIdStore()],
+            ],
+        ]))
+            ->withUnzerKeypair()
+            ->build();
+
+        // Assert
+        $this->expectException(NullValueException::class);
+        $this->expectExceptionMessage('Property "parentIdUnzerCredentials" of transfer `Generated\Shared\Transfer\UnzerCredentialsTransfer` is null.');
```

Comment:
this check is not needed
previous is enough

Reply:
Removed.